src/styles/components/icons/shapes/icon-tiny/styles.less

```diff
@@ -0,0 +1,78 @@
+& when (@icon-tiny-enabled) {
```

Comment:
@nLight yes, that is intentional.  The check is only on the component variant itself, following the pattern set for similar icons in CNVS.